### DISCUSSION TITLE
On the Coupling Method of Using Moose as an Interface

### DISCUSSION THREAD

**Discussion by lvkas521424** [idaholab/moose#27458] [Q&A General]
Hello everyone, as a beginner user and learner of MOOSE, I would like to use MOOSE to provide interfaces to couple two programs with certain computing functions together. Do you have any ready-made cases for me to visit and learn? I cannot find enough tutorials to assist me in learning this knowledge, so I am very confused about how to execute them. Thank you all.

Jiahuilv

**Comment by GiudGiud** [comment]
Hello

Cardinal is an example of an application that compiles external program together with MOOSE
https://github.com/neams-th-coe/cardinal

Guillaume
